lib/iris/fileformats/pp_rules.py

```diff
@@ -968,30 +980,78 @@ def _all_other_rules(f):
 
     # Cross-sectional time values (--> vector coordinates)
     if (len(f.lbcode) == 5 and f.lbcode[-1] == 1 and f.lbcode.iy == 23):
+        epoch_hours_unit = f.time_unit('hours')
         dim_coords_and_dims.append(
             (DimCoord(
                 f.y,
                 standard_name='time',
-                units=Unit('days since 0000-01-01 00:00:00',
+                units=Unit(epoch_hours_unit,
```

Comment:
Then our sample set of previous PP files has not been inclusive enough and bringing new data to the table has necessitated a change to our outlook? As it happens there is plenty wrong with the file in question but according to the data supplier this is not one of them.